src/components/v1/shared/formItems/ProjectPayoutFormItem.tsx

```diff
@@ -0,0 +1,72 @@
+import { BigNumber } from '@ethersproject/bignumber'
+import { t, Trans } from '@lingui/macro'
+import { Form, InputNumber, Radio } from 'antd'
+import { FormItems } from 'components/formItems'
+import {
+  NULL_ALLOCATOR_ADDRESS,
+  V1_V3_ALLOCATOR_ADDRESS,
+} from 'constants/contracts/mainnet/Allocators'
+import { FEATURE_FLAGS } from 'constants/featureFlags'
+import { useState } from 'react'
+import { featureFlagEnabled } from 'utils/featureFlags'
+
+export function ProjectPayoutFormItem({
+  initialHandle,
+  initialAllocator,
+  onChange,
+}: {
+  initialHandle: string | BigNumber | undefined
+  initialAllocator: string | undefined
+  onChange: (id: string) => void
+}) {
+  // need state here (as well as form value in parent component)
+  // to handle the changing of the V1/V3 project input field
+  const [allocator, setAllocator] = useState<string | undefined>(
+    initialAllocator,
+  )
+
+  const allocatorsEnabled = featureFlagEnabled(FEATURE_FLAGS.SPLIT_ALLOCATORS)
+
+  const _V1ProjectHandleFormItem = (
+    <FormItems.ProjectHandleFormItem
+      name="handle"
+      requireState="exists"
+      initialValue={initialHandle}
+      returnValue="id"
+      onValueChange={onChange}
+      formItemProps={{
+        label: t`V1 Project handle`,
+      }}
+      required
+    />
+  )
+
+  if (!allocatorsEnabled) {
+    return _V1ProjectHandleFormItem
+  }
+  return (
+    <>
+      <Form.Item label={t`Project treasury version`} name={'allocator'}>
+        <Radio.Group
+          value={allocator}
+          onChange={e => setAllocator(e.target.value)}
+        >
+          <Radio value={NULL_ALLOCATOR_ADDRESS}>
```

Comment:
**question** does it default to V1 selected? If not can we do that?

Reply:
Yep, it does